Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/toolbar/ToolbarMenuItem.java

```diff
@@ -112,6 +112,22 @@ public void setVisible(boolean visible) {
         }
     }
 
+    public void setEnabled(boolean enabled) {
+        this.enabled = enabled;
+
+        if (view != null) {
+            if (!enabled) {
+                view.setClickable(false);
+                view.setFocusable(false);
+                view.setAlpha(.5f);
```

Comment:
I had it tinting to grey before, because for certain themes, just changing the alpha may make it look sorta bad.

Reply:
Yeah I noticed it too on my real device. But on emulator it works fine, Strange.

Reply:
That's because #FFFFFFFF being tinted grey becomes #FF888888 versus alpha which is #88FFFFFF, which are blended completely differently with the background. Emulator or not the result should be the same.

Reply:
Alright, I will use your implementation with tinting instead of using alpha.

Reply:
Done.